src/wms/src/test/java/org/geoserver/wms/capabilities/GetCapabilitiesRootLayerTitleTest.java

```diff
@@ -0,0 +1,139 @@
+/* (c) 2014 - 2016 Open Source Geospatial Foundation - all rights reserved
```

Comment:
Headers should say 2017 and only to OSGeo, but I'm going to propose a different way to setup the tests.
Can you avoid creating a new test class, we pay a heavy startup price for integration tests, and add checks to the existing CapabilitesIntegrationTest (for WMS 1.3) and CapabilitiesTest (for WMS 1.1).